### DISCUSSION TITLE
My new blog based on your blog template

### DISCUSSION THREAD

**Discussion by lyrx** [vercel/community#5077] [Show & Tell]
### Description

Helloy guys, I just set up my new blog https://www.alexanderweinmann.com/  based on your template https://vercel.com/templates/next.js/tailwind-css-starter-blog . Your template is very cool and feature rich, and I learned a loit when setting it up for my own blog. You have a lot of templates, and as long as you know something about coding for the web they are really a good starter. Let's see if I I will be successfull with this one. I tried my best to fit your standards.

**Comment by lyrx** [comment]
BTW it would be nice to have a template with MetaMask integration or one with several Ethereum gateways maybe. Integration for other P2P related stuff like IPFS, ORBITDB or anything related to SSI (self sovereign identity) would also be nice in my eyes.

**Reply by bulang86** [reply]
你好，莱尔克斯，我想进入你的博客；进不了，显示：未能在“节点”上执行“insertBefore”：要在其前插入新节点的节点不是此节点的子节点。--请教一下，我应该如何操作？
